courseware/api_test.py

```diff
@@ -242,12 +248,47 @@ def test_get_edx_api_client(mocker, settings, user):
 def test_enroll_in_edx_course_runs(mocker, user):
     """Tests that enroll_in_edx_course_runs uses the EdxApi client to enroll in course runs"""
     mock_client = mocker.MagicMock()
+    enroll_return_values = ["result1", "result2"]
+    mock_client.enrollments.create_student_enrollment = mocker.Mock(
+        side_effect=enroll_return_values
+    )
     mocker.patch("courseware.api.get_edx_api_client", return_value=mock_client)
     course_runs = CourseRunFactory.build_batch(2)
-    enroll_in_edx_course_runs(user, course_runs)
-    mock_client.enrollments.create_audit_student_enrollment.assert_any_call(
-        course_runs[0].courseware_id
+    enroll_results = enroll_in_edx_course_runs(user, course_runs)
+    mock_client.enrollments.create_student_enrollment.assert_any_call(
+        course_runs[0].courseware_id, mode=EDX_ENROLLMENT_PRO_MODE
+    )
+    mock_client.enrollments.create_student_enrollment.assert_any_call(
+        course_runs[1].courseware_id, mode=EDX_ENROLLMENT_PRO_MODE
+    )
+    assert enroll_results == enroll_return_values
+
+
+def test_enroll_in_edx_course_runs_audit(mocker, user):
```

Comment:
Is this worth testing for the other error message?

Reply:
Yeah, shouldn't be too hard